src/Neuroblox.jl

```diff
@@ -259,5 +259,6 @@ export powerspectrumplot, powerspectrumplot!, welch_pgram, periodogram, hanning,
 export detect_spikes, mean_firing_rate, firing_rate
 export voltage_timeseries, meanfield_timeseries, state_timeseries, get_neurons, get_exci_neurons, get_inh_neurons, get_neuron_color
 export AdjacencyMatrix, Connector, connection_rule, connection_equations, connection_spike_affects, connection_learning_rules, connection_callbacks
-export inputs, outputs, equations, unknowns, parameters, discrete_events
+export inputs, outputs, equations, unknowns, parameters, discrete_eventse
```

Comment:
I think there's a typo here with `discrete_eventse`
```suggestion
export inputs, outputs, equations, unknowns, parameters, discrete_events
```

Reply:
Good catch! Fixed it now.